client/pong.c

```diff
@@ -424,6 +427,14 @@ void pong_run(Pong* pong) {
       SDL_Delay(TICK_MS);
     }
     else {
+      struct timeval curr_update;
+      gettimeofday(&curr_update, NULL);
+
+      struct timeval difference = timeval_subtract(curr_update, prev_update);
+      if (timeval_compare(&difference, &MIN_UPDATES_TIME_DIFF) == 1) {
```

Comment:
That's not easily readable. It is unclear what the significance of 1 in this context. Could be solved by introducing a function with interface like `bool timeval_less(lhs, rhs)`.